tox.ini

```
@@ -47,3 +47,158 @@ extras= all,tests
 commands=
     pip freeze
     pytest compatibility_tests/ --remote-data
+
+[testenv:asdf-standard]
+changedir={envtmpdir}
+allowlist_externals=
+    git
+commands=
+    git clone https://github.com/asdf-format/asdf-standard.git
+    pip install -e asdf-standard[test]
+    pip freeze
+    pytest asdf-standard
+
+[testenv:asdf-transform-schemas]
+changedir={envtmpdir}
+allowlist_externals=
+    git
+commands=
+    git clone https://github.com/asdf-format/asdf-transform-schemas.git
+    pip install -e asdf-transform-schemas[test]
+    pip freeze
+    pytest asdf-transform-schemas
+
+[testenv:asdf-unit-schemas]
+changedir={envtmpdir}
+allowlist_externals=
+    git
+commands=
+    git clone https://github.com/asdf-format/asdf-unit-schemas.git
+    pip install -e asdf-unit-schemas[test]
+    pip freeze
+    pytest asdf-unit-schemas
+
+[testenv:asdf-wcs-schemas]
+changedir={envtmpdir}
+allowlist_externals=
+    git
+commands=
+    git clone https://github.com/asdf-format/asdf-wcs-schemas.git
+    pip install -e asdf-wcs-schemas[test]
+    pip freeze
+    pytest asdf-wcs-schemas
+
+[testenv:asdf-coordinates-schemas]
+changedir={envtmpdir}
+allowlist_externals=
+    git
+commands=
+    git clone https://github.com/asdf-format/asdf-coordinates-schemas.git
+    pip install -e asdf-coordinates-schemas[test]
+    pip freeze
+    pytest asdf-coordinates-schemas
+
+[testenv:astropy]
+changedir={envtmpdir}
+allowlist_externals=
+    git
+commands=
+    git clone https://github.com/astropy/astropy.git
+    pip install -e astropy[test]
+    pip freeze
+    pytest astropy/astropy/io/misc/asdf
+
+[testenv:asdf-astropy]
+changedir={envtmpdir}
+allowlist_externals=
+    git
+commands=
+    git clone https://github.com/astropy/asdf-astropy.git
+    pip install -e asdf-astropy[test]
+    pip freeze
+    pytest asdf-astropy
+
+[testenv:specutils]
+changedir={envtmpdir}
+allowlist_externals=
+    git
+commands=
+    git clone https://github.com/astropy/specutils.git
+    pip install -e specutils[test]
+    pip freeze
+    pytest specutils
+
+[testenv:gwcs]
+changedir={envtmpdir}
+allowlist_externals=
+    git
+commands=
+    git clone https://github.com/spacetelescope/gwcs.git
+    pip install -e gwcs[test]
+    pip freeze
+    pytest gwcs
+
+[testenv:jwst]
+changedir={envtmpdir}
+allowlist_externals=
+    git
+setenv=
+    CRDS_SERVER_URL = https://jwst-crds.stsci.edu
+    CRDS_PATH = ~/crds_cache
+    CRDS_CLIENT_RETRY_COUNT = 3
+    CRDS_CLIENT_RETRY_DELAY_SECONDS = 20
+commands=
+    git clone https://github.com/spacetelescope/jwst.git
+    pip install -e jwst[test]
+    pip freeze
+    pytest jwst
+
+[testenv:stdatamodels]
+changedir={envtmpdir}
+allowlist_externals=
+    git
```

```suggestion
    git
extras=
```
This would fix the issue with stdatamodels failing because of the included deps from the `extras= all,tests`.

It makes sense to me to have this for all downstream CI tests.